AirCasting/Utils/ButtonStyles/AirBorderedButtonStyle.swift

```diff
@@ -0,0 +1,31 @@
+// Created by Lunar on 08/06/2021.
+//
+
+import SwiftUI
+
+struct AirBorderedButtonStyle: ButtonStyle {
```

Comment:
can you explain what exactly is `airboarded`?

Reply:
I couldn't use` BorderedButtonStyle()` because it already existed. I created a custom button style, and like in case of SectionPicker I added "air" at the beginning which stands for AirCasting.

Reply:
🤔 imo not a good approach to prefix with air. Same would be if You added `Bordered2ButtonStyle` etc etc. says nothing. @anna1901 @pawelgil @jankrzempek any thoughts on how could we handle the naming convention here? 

[interesing](https://pspdfkit.com/blog/2018/naming-classes-why-it-matters-how-to-do-it-well/)

Reply:
Sure, it would be great to come up with some naming convention rules :)

Reply:
@pawelgil

Reply:
tough. `BorderedButtonStyle` is perfect. In my opinion this kind of a naming collision is a clear code smell indicating that we need a namespace for our own stylings. To me it looks like the desired way to achieve namespacing in this case would be by creating a swift module for all in-house button styles (or perhals all styling, not only buttons). It's really lightweight and easy to do. We'd have a module called let's say `AirCastingStyling` the sake of this argument, and than in the codebase:
```Swift
import AirCastingStyling

...
.buttonStyle(AirCastingStyling.BorderedButtonStyle(...))
...
```

If not, we're basically in a position where we need to compromise and, conceptually, I think `Air` prefix is as good as any because the best fitting option `BorderedButtonStyle` is off the table anyway and we need to bandaid.